tests/common/plugins/psu_controller/snmp_psu_controllers.py

```diff
@@ -89,16 +150,21 @@ def turn_on_psu(self, psu_id):
         @param psu_id: ID of the PSU on SONiC DUT
         @return: Return true if successfully execute the command for turning on power. Otherwise return False.
         """
-        try:
-            idx = int(psu_id) % len(self.pdu_ports)
-            port_oid = self.PORT_CONTROL_BASE_OID + self.pdu_ports[idx]
-            cmd = "snmpset -v1 -C q -c private %s %s i %s" % (self.controller, port_oid, self.CONTROL_ON)
-            run_local_cmd(cmd)
-            logging.info("Turned on PSU %s" % str(psu_id))
+        idx = int(psu_id) % len(self.pdu_ports)
```

Comment:
if you have the mapping between the psu_id of a dut to the psu_ports, then we do not need to guess here.

Reply:
fixed this.

Reply:
Will fix this with separate PR.